3469/1/program.rb

```diff
@@ -0,0 +1,35 @@
+class TopGems
+  include IoStream
+  def initialize(key)
```

Comment:
It is bad practice to return value from initialize, you should add another public method